config.py

```diff
@@ -1,6 +1,7 @@
 class DefaultConfig(object):
     DEBUG = True
     CSRF_ENABLED = True
-    SECRET_KEY = '[\xd7\xda\x0cZ\x8a8\x8b\xb1g~\x0b\x86\x96\xf5\xe3\xa5Q\xfb\x86\x03\xb2\xfc'
-    SQLALCHEMY_DATABASE_URI = 'sqlite:////home/caterina/node_1.db'
-    SQLALCHEMY_TRACK_MODIFICATIONS = False
\ No newline at end of file
+    SECRET_KEY = 'app secret key'
```

Comment:
I hope you generated a different one, as the deleted one will stay here forever.